index.html

```diff
@@ -2948,7 +2941,8 @@ <h3>Receiving and parsing content</h3>
               <li>
                 Let <var>MIME type</var>.parameters[<code>"lang"</code>] be the
                 result of invoking <a>ASCII decode</a> on the consecutive
-                <var>offset</var> number of <a>bytes</a>.
+                <var>offset</var> number of <a>bytes</a>
+                (From second <a>byte</a> and <var>offset</var> <a>bytes</a> ahead).
```

Comment:
You want something like from the second to the offset + 1 byte, inclusive, of ndef.PAYLOAD, no?

Reply:
yes. Everything shifted by 1 byte

Reply:
To my understanding, from the current "pointer" counting ```offset``` number of bytes, i.e. including the first byte.

Reply:
Fixed